File: deaf_grandma.py
def deaf_grandma():
    count_bye = 0
    print('HEY KID!')
    while count_bye < 2:

        kid_rep = input('Input a reply to grandma.')
        if kid_rep == "":
            print ('WHAT?!')
            
        elif kid_rep == 'GOODBYE!':
            count_bye +=1
            if count_bye == 2:
                print('LATER, SKATER!')
            else:
                print ('LEAVING SO SOON?')

        elif kid_rep.isupper():
            print('NO, NOT SINCE 1946!')
        elif kid_rep.islower():
            print ("SPEAK UP, KID!")
        





deaf_grandma()

File: deaf_grandma.js
function deafGrandma() {
    let count_bye = 0;
    alert('HEY, KID!')

    while (count_bye < 2) {
        let kid_rep = window.prompt('Input a reply to grandma.')
        if (kid_rep == "") {
            alert('WHAT?!')
        } else if (kid_rep == 'GOODBYE!') {
            count_bye += 1
            if (count_bye === 2) {
                alert('LATER, SKATER!')
                return
            } else {
                alert('LEAVING SO SOON?')
            }
        } else if (kid_rep === kid_rep.toUpperCase()) {
            alert('NO, NOT SINCE 1946!')
        } else if (kid_rep === kid_rep.toLowerCase()) {
            alert('SPEAK UP KID!')
        }
    }

}


deafGrandma();